certified-connectors/Cloudspark EXP Leave/apiDefinition.swagger.json

```diff
@@ -0,0 +1,534 @@
+              "title": "Start Date"
+            },
+            "hours": {
+              "type": "string",
+              "description": "Number of hours requested.",
+              "title": "Hours"
+            },
+            "description": {
+              "type": "string",
+              "description": "User supplied description about the time off request.",
+              "title": "Description"
+            },
+            "userId": {
+              "type": "string",
+              "description": "ID of user submitting a time off request.",
+              "title": "User ID"
+            },
+            "policyName": {
+              "type": "string",
+              "description": "Type of time off requested.",
+              "title": "Policy Type"
+            },
+            "leaveId": {
+              "type": "string",
+              "description": "ID of the time off request.",
+              "title": "Leave ID"
+            },
+            "displayName": {
+              "type": "string",
+              "description": "User's full name.",
+              "title": "User Name"
+            },
+            "companyName": {
+              "type": "string",
+              "description": "Name of user's company.",
+              "title": "Company Name"
+            },
+            "EncodedImageContent": {
+              "type": "string",
+              "description": "User's profile image.",
+              "title": "User Image"
+            },
+            "mail": {
+              "type": "string",
+              "description": "User's email address.",
+              "title": "User Email"
+            }
+          }
+        }
+      },
+      "post": {
+        "responses": {
+          "200": {
+            "description": "200",
+            "schema": {
+              "description": "Response",
+              "title": "TriggerResponse",
+              "type": "string"
+            }
+          },
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Trigger the workflow when a time off request is submitted",
+        "description": "Leave Submitted Trigger",
+        "operationId": "LeaveSubmitted",
+        "consumes": [
+          "application/json"
+        ],
+        "x-ms-trigger": "single",
+        "parameters": [
+          {
+            "name": "tenant_id",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "description": "Paste in your Tenant ID",
+            "x-ms-summary": "Tenant ID",
+            "x-ms-visibility": "important"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "url": {
+                  "type": "string",
+                  "description": "url",
+                  "title": "",
```

Comment:
@DinuWije Empty properties can be either removed or provided with valid values

Reply:
Resolved (provided valid values)